main.tex

```diff
@@ -599,22 +602,60 @@
+\item An Owner or Occupier of a lot is entitled to enjoyment of the Common Property in ways that shall hereby be know as Reasonable Uses. These include:
+\begin{enumerate}[label=(\alph*)]
+    \item A place for private conversation
+    \item A place to breathe reasonably fresh air, unpolluted by smoking (see the section on smoking)
+
+\end{enumerate}
+\item For outdoor Common Property, an Owner or Occupier of a lot is entitled to enjoyment of further activities, including:
+\begin{enumerate}[label=(\alph*)]
+    \item A place for pets to explore (while physically restrained by a human)
+    \item A place to sit
+    \item A place to lie down
+    \item A place to play background music. For this purpose, a mini portable speaker can be brought into the central courtyard. For the purpose of this Rule, ‘background music’ means music at a level that enables Owners, Occupiers or invitees to conduct an uninterrupted or undisturbed conversation at a distance of 3 metres from each other and within 3 metres of the speaker, within the central courtyard.
+    \item A place to breathe reasonably clean air, free from odours of smoke, garbage or biological excrement.
+\end{enumerate}
+\item Some activities which are possible in the Common Property would necessarily require the exclusion of other Lot Owners and Occupiers, but no Lot Owners or Occupiers shall be required to cooperate in maintaining a group's exclusive enjoyment of the Common Property. Such exclusive behaviours include:
+\begin{enumerate}[label=(\alph*)]
+    \item Meditation
+    \item Religious services
+    \item Ceremonies
+    \item Sports and games requiring non-stationary participation.
+    \item The use of recreational transportation, such as roller skates and skateboards.
+    \item Playing music
+\end{enumerate}
+
+\item Some activities in the Common Property would necessarily cause unreasonable interference with other Lot Owners or Occupiers and are not permitted. Such unreasonable behaviours include:
+\begin{enumerate}[label=(\alph*)]
+    \item Shouting
+    \item Barking
+    \item Playing music loud enough that it's audible to humans, dogs or cats through closed windows.
+    \item Littering
+    \item Human defecation / urination
+    \item Non-human defecation / urination (without immediate cleaning)
+    \item Spitting
+    \item Digging holes
+    \item Cutting the plants of the Common Property
+    \item Picking flowers from the Common Property
+    \item Leaving children unattended
+\end{enumerate}
 
-\item  An Owner or Occupier of a Lot must not without the written approval of the Owners Corporation use for his or her own purposes as a garden or for any other purpose, any portion of the Common Property. Any approval may state a period for which the approval is granted.
```

Comment:
Can't use the common property for a garden or for "any other purpose"

**Any other purpose**.